src/plugins/orangehrmCorporateBrandingPlugin/test/Api/ThemeAPITest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA
+ */
+
+namespace OrangeHRM\Tests\CorporateBranding\Api;
+
+use OrangeHRM\CorporateBranding\Api\ThemeAPI;
+use OrangeHRM\CorporateBranding\Service\ThemeService;
+use OrangeHRM\Entity\Theme;
+use OrangeHRM\ORM\Doctrine;
+use OrangeHRM\Tests\Util\EndpointIntegrationTestCase;
+use OrangeHRM\Tests\Util\Integration\TestCaseParams;
+
+/**
+ * @group CorporateBranding
+ * @group APIv2
+ */
+class ThemeAPITest extends EndpointIntegrationTestCase
+{
+    /**
+     * @dataProvider dataProviderForTestGetOne
+     */
+    public function testGetOne(TestCaseParams $testCaseParams): void
+    {
+        $this->populateFixtures('ThemeAPI.yaml');
+
+        $this->registerServices($testCaseParams);
+        $api = $this->getApiEndpointMock(ThemeAPI::class, $testCaseParams);
+        $this->assertValidTestCase($api, 'getOne', $testCaseParams);
+    }
+
+    /**
+     * @return array
+     */
+    public function dataProviderForTestGetOne(): array
+    {
+        return $this->getTestCases('ThemeAPITestCases.yaml', 'GetOne');
+    }
+
+    public static function keepOnlyDefaultThemePreHook(): void
```

Comment:
Is this function being used?

Reply:
yes,
https://github.com/orangehrm/orangehrm/blob/a1c5c5e7bf9ef6fa0a9b0bc5edd4245a07174e27/src/plugins/orangehrmCorporateBrandingPlugin/test/fixtures/testcases/ThemeAPITestCases.yaml#L32